grails-app/controllers/org/broadinstitute/orsp/SearchController.groovy

```diff
@@ -195,7 +195,7 @@ class SearchController implements UserInfo {
         if (params.dateRestriction) options.dateRestriction = Date.parse('mm/dd/yyyy', params.dateRestriction)
         if (params.methodsResearchExcluded) options.methodsResearchExcluded = true
         List<Issue> issues = queryService.findByQueryOptions(options)
-        render (view: '_dataUseRecords', model: ["issues": issues])
+        render(view: '_dataUseRecords', model: ["issues": issues])
```

Comment:
All changes to this class should be reverted

Reply:
done